google-datacatalog-kafka-connector/src/google/datacatalog_connectors/kafka/config/tag_templates_constants.py

```diff
@@ -0,0 +1,63 @@
+from google.cloud import datacatalog_v1beta1
+
+
+class TagTemplateConstants:
+    """
+    Constants from this class are used both by
+    prepare.DataCatalogTagTemplateFactory and by
+    respective tests. This class is intended to be the
+    source of truth regarding which fields are expected to exist
+    in the tag templates.
+    """
+
+    DISPLAY_NAME_IDX = 0
+    FIELD_TYPE_IDX = 1
```

Comment:
Any reason for using `IDX` and not just `ID` ?

Reply:
Usually I'd map `Identificator` to `ID` and `Index` to `IDX`. So since here I mean `Index`, I named it `XYZ_IDX`.